src/unistd.rs

```diff
@@ -3355,6 +3355,55 @@ mod getres {
     }
 }
 
+feature! {
+#![feature = "process"]
+#[cfg(target_os = "freebsd")]
+pub mod rfork_freebsd {
```

Comment:
We don't need to put it in a dedicated module, the existing 2 `pub mod` within `unistd.rs` are for better naming

* acct: https://github.com/nix-rust/nix/pull/952#discussion_r224821858
* alarm: https://github.com/nix-rust/nix/pull/830#discussion_r164946670

`rfork(2)`, IMHO, is good enough so we don't need this